api/v1/handlers.go

```diff
@@ -1794,13 +1795,19 @@ func (s *apiV1) handleAdminGetMiners(c echo.Context) error {
 		out[i].Name = m.Name
 		out[i].Version = m.Version
 
-		ci, err := s.minerManager.GetMinerChainInfo(ctx, m.Address.Addr)
-		if err != nil {
-			out[i].ChainInfo = ci
-		}
-
+		// Spawn a thread to fetch the Chain Info (Lotus RPC call - takes a few ms)
+		wg.Add(1)
+		go func(w *sync.WaitGroup, addr address.Address, i int) {
```

Comment:
This will spawn a new go thread per Active Storage Provider in our db. Currently about  ~100 in prod. Each call only takes a few ms to complete.

Reply:
Great stuff man! Thanks for this.

One request-  can we add a verification test in this PR? Just a screenshot of postman response when calling `handleAdminGetMiners` endpoint.

Thank you!